packages/cashier/package.json

```diff
@@ -93,5 +93,8 @@
         "webpack-cli": "^4.7.2",
         "webpack-manifest-plugin": "^4.0.2",
         "webpack-node-externals": "^2.5.2"
+    },
+    "peerDependencies": {
+        "@deriv/core": "0.1.0"
```

Comment:
Since `@deriv/core` already exists in the `node_modules`, We can add it as `peerDependencies` in other packages to access the `core` package, And since we are using it for only types there won’t be any issues in building the project.